net/acme/files/acme.config

```diff
@@ -3,14 +3,31 @@ config acme
        option account_email 'email@example.org'
        option debug 0
 
+config cert 'example_wildcard'
+       option enabled 0
+       option use_staging 1
+       option keylength 2048
+       option update_uhttpd 1
+       option update_nginx 1
+       list domains example.org
+       list domains sub.example.org
+       list domains *.sub.example.org
+       # option user_setup "path-to-custom-setup.script"
+       # option user_cleanup "path-to-custom-cleanup.script"
+       option dns "dns_freedns"
+       list credentials 'FREEDNS_User="ssladmin@example.org"'
+       list credentials 'FREEDNS_Password="1234"'
+       option calias "exampleorg-acmechallenge.mooo.com"
+       option dalias "exampleorg-acmechallenge.mooo.com"
```

Comment:
Let's keep this to example.TLD instead of squatting on something that could technically be someone's domain. You can just use example.com as the distinct other domain :)

Reply:
Good point. I have updated it.

Technically a domain alias should point to a sub domain due to reasons (the docs aren't clear to me... it's either an acme API limitation or an acme.sh DNS API limitation). Sometimes it works to use ".example.com" or "@.example.com". Since run.sh does nothing to try and make it work I just used "dalias.example.com" rather than give a potential problematic example.